.travis.yml

```diff
@@ -1,28 +1,47 @@
 language: cpp
+os:
+ - linux
 compiler:
 - clang
 - gcc
+- wcl
 rvm:
 - 1.9.3
 env:
-  matrix:
   - BUILDTOOL=autotools
+  - BUILDTOOL=autotools_old_compilers
   - BUILDTOOL=cmake
   - BUILDTOOL=cmake-coverage
-#  - BUILDTOOL=autotools_old_compilers
-  global:
-  - secure: |-
+  - BUILDTOOL=make-dos
+matrix:
+    exclude:
+    - env: BUILDTOOL=cmake-coverage
+    - env: BUILDTOOL=autotools_old_compilers
+    - env: BUILDTOOL=make-dos
+    - compiler: wcl
+    include:
+    - env: BUILDTOOL=cmake-coverage
+      compiler: gcc
+    - env: BUILDTOOL=make-dos
+      compiler: wcl
+global:
+- secure: |-
       P05xUfJVw5YM4hF7hzQLjyMzDD4Q1/fyWP9Uk5aK5VrSWNY99EuxldXI5QK/
       vA1NkcW49tQW1wQvBlRtdlLNOmUfDP/oiJFXPwNn4dqwOIOEet2P7JO/5hnH
       MNHlZmGu2WpoZREhOFBfsIhK0IP8mloqLDq2XemBdga/LWygrLU=
-  - secure: Y/8iNkf6uEbE3qltnM+7mGlCvFWzyttwwRGgVGw1m9xOiUJcobvOImQRU8XZ91dgO+Fz0A3mljqs1sK1OPjpXmFGE1jP/NlotMw0WlDOuSIDjQ4ubwdTNGAwNY53R9ygbIjEmqxHAJm9mOZqxW2hNaoI7TcX6oX248/hLibyx8M=
+- secure: Y/8iNkf6uEbE3qltnM+7mGlCvFWzyttwwRGgVGw1m9xOiUJcobvOImQRU8XZ91dgO+Fz0A3mljqs1sK1OPjpXmFGE1jP/NlotMw0WlDOuSIDjQ4ubwdTNGAwNY53R9ygbIjEmqxHAJm9mOZqxW2hNaoI7TcX6oX248/hLibyx8M=
 before_install:
 - sudo pip install cpp-coveralls
+- sudo apt-get install dosbox -y
 install:
 - gem install travis_github_deployer
 - sudo apt-get update --fix-missing
 - sudo apt-get install valgrind
 before_script:
+- wget ftp://ftp.openwatcom.org/pub/open-watcom-c-linux-1.9 -O /tmp/watcom.zip
```

Comment:
Why not move this inside the travis build script? I think you misunderstood this earlier. Now it will be on all targets, right? But if you add it to the travis_ci_build.sh file, then you can specify it for only the Watcom build.